### DISCUSSION TITLE
If I have a non-mandatory schema based options picker, how does the user delete/remove selection

### DISCUSSION THREAD

**Discussion by kpturner** [Budibase/budibase#11116] [Help]
Let's say I create a schema based options picker for a field that is not mandatory.  When the selects an option and hits save, that option is persisted.  Now the user comes along and no longer wants a value for that field.  Usually there is scope to select a blank or null value from the list but for the BB options picker there is no such option that I can see. Once you've picked something, you are stuck with it.  What is the normal way to achieve this?  Have I got to add a `delete` icon against every optional picker I have created?

**Comment by kpturner** [comment]
More context.   I have two such pickers. There is no difference apart from the name of the field as far as I can see. 
With one, the user is able to deselect by clicking on the top row of the select - which contains the placeholder text:
![image](https://github.com/Budibase/budibase/assets/308646/88f0e2df-0cad-4928-ad41-da5a0e38dc5d)
With the other, the placeholder text is not an option
![image](https://github.com/Budibase/budibase/assets/308646/79ee1389-be4a-4b2f-9fb8-ac4a39fa5424)

**Reply by melohagan** [reply]
Hey @kpturner this is curious. Are you able to share any more screenshots please? I would like to see your component tree and the table/form schemas here.

Then I can see about reproducing.

**Comment by emerson-nascimento** [comment]
The same with me. I can not set to null value even direct in the database. You really have to mandatory select one option even if you dont need more. Maybe I resolve this with sql query, but really frusted on this troubble.

**Reply by melohagan** [reply]
Hey @emerson-nascimento Are you able to use the **Update Field Value** action and select *Reset to default* ?
